Dockerfile

```diff
@@ -1,15 +1,66 @@
+FROM ubuntu:25.04 AS selenium-manager-image
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    ca-certificates curl jq wget \
+  && rm -rf /var/lib/apt/lists/*
+
+ARG SELENIUM_MANAGER_VERSION=latest
+RUN SELENIUM_MANAGER_VERSION_FULL=$(curl -s https://api.github.com/repos/SeleniumHQ/selenium_manager_artifacts/releases/${SELENIUM_MANAGER_VERSION} | jq -r .name) \
+  && wget --no-verbose https://github.com/SeleniumHQ/selenium_manager_artifacts/releases/download/${SELENIUM_MANAGER_VERSION_FULL}/selenium-manager-linux \
+  && chmod +x selenium-manager-linux \
+  && mv selenium-manager-linux /usr/local/bin/selenium-manager
+
+ARG CHROME_VERSION="latest"
+ARG FIREFOX_VERSION="latest"
+# Note: geckodriver version needs to be updated manually
+ARG GECKODRIVER_VERSION="0.34.0"
+
+RUN if [ $FIREFOX_VERSION = "latest" ]; then SE_FIREFOX_VERSION="stable"; \
+  elif [ $FIREFOX_VERSION = "nightly-latest" ]; then SE_FIREFOX_VERSION="nightly"; \
+  elif [ $FIREFOX_VERSION = "devedition-latest" ]; then SE_FIREFOX_VERSION="dev"; \
+  elif [ $FIREFOX_VERSION = "esr-latest" ]; then SE_FIREFOX_VERSION="esr"; \
```

Comment:
It is nice to have these options, but I don't think we would utilize them. I think just two options (latest or specifying the version number) should be enough for us (same as chrome)

Reply:
Applied the changes. Thanks! - 081e92223f2dd4fd50f299b3df3db8664c827925